configure

```diff
@@ -944,6 +946,8 @@ Optional Libraries:
   --with-opengl-prefix=DIR Prefix where OpenGL (ES) is installed (optional)
   --disable-opengl         disable OpenGL (ES) support [autodetect]
 
+  --with-dispmanx          enable dispmanx (Raspberry Pi native 2D API) rendering 
```

Comment:
This was removed. Sorry, of course it made no sense. It's handled by --enable-dispmanx as you supposed.